.github/workflows/ci.yml

```diff
@@ -34,62 +34,64 @@ jobs:
-  #         rm sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip
-  #         dnf install -y java-11-openjdk
-  #
-  #     - name: Checkout sources
-  #       uses: actions/checkout@v1
-  #
-  #     # In a PR use pre-built 3rd parties to optimize the duration of the job
-  #     - name: Build branch PR
-  #       env:
-  #         CONTAINER_WORKSPACE: ${{ runner.workspace }}
-  #         DYNAWO_THIRD_PARTY_INSTALL_DIR: /opt/dynawo/3rParty
-  #       if: github.ref != 'refs/heads/master' && github.event_name == 'push'
-  #       run: |
-  #         export DYNAWO_HOME=$(pwd)
-  #         $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests-coverage
-  #
-  #     - name: Build master
-  #       env:
-  #         CONTAINER_WORKSPACE: ${{ runner.workspace }}
-  #       if: github.ref == 'refs/heads/master' && github.event_name == 'push'
-  #       run: |
-  #         export DYNAWO_HOME=$(pwd)
-  #         util/envDynawo.sh build-3rd-party
-  #         $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests-coverage
-  #
+  coverage:
+    runs-on: ubuntu-latest
+    container: dynawo/dynawo-ci:latest
+    env:
+      DYNAWO_CXX11_ENABLED: YES
+      DYNAWO_COMPILER: GCC
+      DYNAWO_RESULTS_SHOW: "false"
+      DYNAWO_INSTALL_OPENMODELICA: /opt/OpenModelica/Install
+      DYNAWO_SRC_OPENMODELICA: /opt/OpenModelica/Source
+      DYNAWO_NB_PROCESSORS_USED: 2
+    steps:
+      - name: Install Sonar wrapper
+        working-directory: ${{ runner.workspace }}
+        env:
+          CONTAINER_WORKSPACE: ${{ runner.workspace }}
+        run: |
+          wget https://sonarcloud.io/static/cpp/build-wrapper-linux-x86.zip
+          unzip build-wrapper-linux-x86.zip
+
+      - name: Install Sonar scanner
+        working-directory: ${{ runner.workspace }}
+        env:
+          SONAR_SCANNER_VERSION: 4.5.0.2216
+        run: |
+          wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip
+          unzip sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip
+          ln -s sonar-scanner-${SONAR_SCANNER_VERSION} sonar
+          rm sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip
+          dnf install -y java-11-openjdk
+
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      # In a PR use pre-built 3rd parties to optimize the duration of the job
+      # For now, run only tests, not coverage
+      - name: Build branch PR
+        env:
+          CONTAINER_WORKSPACE: ${{ runner.workspace }}
+          DYNAWO_THIRD_PARTY_INSTALL_DIR: /opt/dynawo/3rParty
+        if: github.ref != 'refs/heads/master' && github.event_name == 'push'
+        run: |
+          export DYNAWO_HOME=$(pwd)
+          $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests
```

Comment:
comment line and use only util/envDynawo.sh build-tests, wrapper is for sonar

Reply:
OK